spec/formatters/connect

```diff
@@ -6,7 +6,7 @@ store Test {
 }
 
 component A {
-  connect Test exposing{a,b,c,d}
+  connectTest exposing{a,b,c,d}
```

Comment:
this doesnt seem right

Reply:
Strictly speaking the parser does not need white-space between the keyword and the name of the store here. 

It's because there is no tokeniser phase, the parser directly produces the AST.